src/modules/ToDoList.js

```diff
@@ -0,0 +1,116 @@
+let List = JSON.parse(localStorage.getItem('tasks')) || [];
```

Comment:
I suggest you to have a single `module` folder containing all exported modules, I suggest you read this article [module](https://github.com/Mukaba/js-good-practice/pull/1) if you need a big picture about ES6 modules.

Reply:
thank you for your suggestions.